docs/cli.md

```diff
@@ -648,29 +648,6 @@ poetry run my-script
 
 Note that this command has no option.
 
-## shell
```

Comment:
Maybe, we should not remove it (now) but write that it has been outsourced into a plugin and we recommend to use `env activate` instead. (We did not remove `export` from the docs. I hope this will save us some support requests because some people will look here.)